en/ros2/user_guide.md

```diff
@@ -264,6 +267,13 @@ To create and build the workspace:
    git clone https://github.com/PX4/px4_ros_com.git
    ```
 
+1. **(Optional)** From PX4 v1.16 (main), include the [Message Translation Node](../ros2/px4_ros2_msg_translation_node.md) into your workspace by running the following script.
+   This step is needed only if using `px4_msgs` that do not match your target version of PX4.
```

Comment:
I modified this line. Please check it is correct.

FWIW I think the updates to this doc work well.